main/Smartsheet/Api/Internal/Util/PathAndQueryWrapperBuilder.cs

```diff
@@ -0,0 +1,121 @@
+﻿//    #[license]
```

Comment:
As mentioned previously there are a couple methods that can be removed from this class. Let's name the class the same as in Java and implement the same methods. See: https://github.com/smartsheet-platform/smartsheet-java-sdk/blob/release-2.0/src/main/java/com/smartsheet/api/internal/util/QueryUtil.java